### PR TITLE
adding CVE entry for mistune<0.8.1

### PR DESCRIPTION
**Pull request by rohit-jamuar**
https://github.com/pyupio/safety-db/issues/2269

@jayfk

### CODE REVIEW

**Comment by rohit-jamuar** [comment]
I'm unsure of how the test-failure on CI is related to this patch. Can one of the core-devs suggest a workaround / fix?

File: data/insecure_full.json [hunk 1]
```diff
@@ -3897,6 +3897,15 @@
                 "<0.7.2"
             ],
             "v": "<0.7.2"
+        },
+        {
+            "advisory": "Cross-site scripting (XSS) vulnerability in the _keyify function in mistune.py in Mistune before 0.8.1 allows remote attackers to inject arbitrary web script or HTML by leveraging failure to escape the 'key' argument.",
+            "cve": 'CVE-2017-16876',
```
Comment (whisller) [inline data/insecure_full.json]: @rohit-jamuar I assume it's failing because of `'`, it's not valid format in JSON

**Comment by rohit-jamuar** [comment]
@whisller Thanks, that was it.

**Comment by Jwomers** [comment]
This will be added to safety-db when our monthly sync happens from PyUps vulnerability db. Thanks for the input!

**Pull request closed without merging**